pkg/gui/keybindings.go

```diff
@@ -10,170 +9,19 @@ import (
 	"github.com/jesseduffield/lazygit/pkg/constants"
 	"github.com/jesseduffield/lazygit/pkg/gui/context"
 	"github.com/jesseduffield/lazygit/pkg/gui/controllers/helpers"
+	"github.com/jesseduffield/lazygit/pkg/gui/keybindings"
 	"github.com/jesseduffield/lazygit/pkg/gui/types"
 )
 
-var keyMapReversed = map[gocui.Key]string{
```

Comment:
had to move this out to prevent a circular dependency but it also makes sense being split out